assets/app/views/directives/osc-key-values.html

```diff
@@ -16,7 +16,7 @@
           <input class="form-control"
                  type="text"
                  name="value"
-                 placeholder="Value"
+                 placeholder="{{valueTitle}}"
```

Comment:
Can you try `ng-attr-placeholder` to prevent `{{valueTitle}}` from ever flickering in the field briefly?